src/DecompBag/DecompBag.js

```diff
@@ -126,97 +129,10 @@ const DecompBag = () => {
         }
       };
 
-      const freshRecords = await fetchRecords(freshBiomass_url);
-      const dryRecords = await fetchRecords(dryBiomass_url);
-      const cnRecords = await fetchRecords(cnBiomass_url);
-      const ashRecords = await fetchRecords(ashBiomass_url);
-
-      const leftJoin = (objArr1, objArr2, key1, key2, key3) =>
-        objArr1.map((obj1) => ({
-          ...objArr2.find(
-            (obj2) =>
-              obj1[key1] === obj2[key1] &&
-              obj1[key2] === obj2[key2] &&
-              obj1[key3] === obj2[key3]
-          ),
-          ...obj1,
-        }));
-
-      const freshDryJoined = leftJoin(
-        freshRecords,
-        dryRecords,
-        "code",
-        "subplot",
-        "subsample"
-      );
-      const cnAshJoined = leftJoin(
-        cnRecords,
-        ashRecords,
-        "code",
-        "subplot",
-        "subsample"
-      );
-      const allJoined = leftJoin(
-        freshDryJoined,
-        cnAshJoined,
-        "code",
-        "subplot",
-        "subsample"
-      );
-
-      //   const freshDryJoined = freshRecords.map(function (e) {
-      //     return Object.assign(
-      //       {},
-      //       e,
-      //       dryRecords.reduce(function (acc, val) {
-      //         if (
-      //           val.code === e.code &&
-      //           val.subplot === e.subplot &&
-      //           val.subsample === e.subsample
-      //         ) {
-      //           return val;
-      //         } else {
-      //           return acc;
-      //         }
-      //       }, {})
-      //     );
-      //   });
+      const bagRecords = await fetchRecords(decompBag_url);
 
-      //   const cnAshJoined = cnRecords.map(function (e) {
-      //     return Object.assign(
-      //       {},
-      //       e,
-      //       ashRecords.reduce(function (acc, val) {
-      //         if (
-      //           val.code === e.code &&
-      //           val.subplot === e.subplot &&
-      //           val.subsample === e.subsample
-      //         ) {
-      //           return val;
-      //         } else {
-      //           return acc;
-      //         }
-      //       }, {})
-      //     );
-      //   });
+      const allJoined = bagRecords;
```

Comment:
is there a reason for assigning `allJoined = bagRecords` instead of `const allJoined = await fetchRecords(decompBag_url);`?

Reply:
> 

No nothing in particular. Thought to keep it just like it was earlier for other left join tables. I can update it to const allJoined = await fetchRecords(decompBag_url);

Reply:
yeah please make it just one variable instead of creating a new one. i'll merge after that

Reply:
Done.